package.json

```diff
@@ -29,17 +29,17 @@
   ],
   "main": "./dist/main.js",
   "bin": {
-    "tact": "./bin/tact"
+    "tact": "./bin/tact.mjs"
```

Comment:
Is there really no way to keep the `tact` name? It feels like having to call it `tact.msj` from command line makes UX a bit worse

Reply:
The end user won't call it `tact.mjs` — the name will stay `tact`, as is.

See [this](https://github.com/novusnota/tact-extracted-ls/blob/e273d453e2642aa2b900c19fc82bd53ddbc3cfdf/package.json#L28) for example:
![image](https://github.com/tact-lang/tact/assets/68142933/0d837827-ea3d-40f7-9eb5-e6f5b2827174)

Despite that the actual bin script there is called `bundle.js`, the user, who downloads this package from npm, receives only `tact-extracted-ls` :)

More details: https://docs.npmjs.com/cli/v9/configuring-npm/package-json#bin

Reply:
Got it, thanks @novusnota. We then should fix it in CI

Reply:
So, long story short, we can keep calling it `tact` but the code will be a tad more convulted (basically the cli code would live under `import('meow').then(() => {/* cli code here*/});` since `meow` is an ES module and `tact` is CommonJS. 

If that trade off is OK, I can rename it back to `tact`.

Reply:
@vitorpy it's probably best to keep things simple, see: https://github.com/tact-lang/tact/pull/287#discussion_r1582045728

Reply:
@vitorpy Right, the script's name in the repo does not really matter as long as there is a way to call it `tact` from the command line. And in this case it will be accessible after a local package installation as @novusnota mentioned